x-pack/plugin/inference/build.gradle

```diff
@@ -27,6 +27,7 @@ base {
 
 dependencies {
   implementation project(path: ':libs:elasticsearch-logging')
+  testImplementation project(path: ':test:framework')
```

Comment:
Why was this added? This should be automatically added to all projects. Plus this PR is just adding a new QA project.

Reply:
I'll try removing it